enabler/src/components/payment-methods/klarna-pay-later.ts

```diff
@@ -19,7 +19,7 @@ import {
 export class KlarnaPayLaterBuilder extends AdyenBaseComponentBuilder {
 
   constructor(baseOptions: BaseOptions) {
-    super(PaymentMethod.klarna, baseOptions);
+    super(PaymentMethod.klarna_pay_later, baseOptions);
```

Comment:
We do need to take into account that existing klarna configured payment methods might need to be removed and re-added. For example on [this one](https://ca-dev.mc-preview.europe-west1.gcp.escemo.com/dev-commercetools-checkout/commercetools-checkout/applications/5c47ed74-8e66-4c7c-b8a8-ae3b585e978f/payment-methods-connect/d661c614-a979-4f82-84c4-7d25b4b142d2) it says `klarna_paynow` but I am not 100% if this will still work and we are backwards compatible.

Not a real problem since we are the only ones who have configured klarna payment methods so we can easily remove and re-add those ones.